plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -40,7 +40,7 @@ class ContainerServiceFormSection extends Component {
       <span>
         {'If your service requires additional files and/or archives of files, enter their URIs to download and, if necessary, extract these resources. '}
         <a
-          href="https://mesosphere.github.io/marathon/docs/application-basics.html"
+          href="https://mesosphere.github.io/marathon/docs/artifact-store.html"
```

Comment:
This is not the correct link. It's about the Artifact Store and doesn't detail how to configure `fetch` (aka. `uris`). Please use `https://mesosphere.github.io/marathon/docs/application-basics.html` which at least touches the matter. 

@sascala do we have a better doc somewhere than the general app config doc?